packages/blockchain/src/db/helpers.ts

```diff
@@ -43,7 +44,7 @@ function DBSetBlockOrHeader(blockBody: Block | BlockHeader): DBOp[] {
     isGenesis ||
     (blockBody instanceof Block && (blockBody.transactions.length || blockBody.uncleHeaders.length))
   ) {
-    const bodyValue = rlp.encode(blockBody.raw().slice(1))
+    const bodyValue = Buffer.from(RLP.encode(bufArrToArr(blockBody.raw()).slice(1)))
```

Comment:
It's something we've talked about at length. I think it's the direction we want to go, but all our APIs support Buffers so it would take a lot of work to update, probably a similar effort level of the BN.js->bigint change. Buffers use Uint8Arrays  as their data type so I don't believe it's a big deal to make a Buffer out of a Uint8Array. We could do some benchmarking to confirm that it is relatively quick. In general we can move away from Buffers because they provide very little added benefit as opposed to just working with Uint8Arrays directly, as we've done with `rlp` and as `js-ethereum-cryptography` has done switching primarily from Buffers to Uint8Arrays. Also gives us broader ecosystem support since you have to polyfill Buffer in browser and other contexts.